handsontable/src/plugins/dropdownMenu/dropdownMenu.js

```diff
@@ -452,7 +453,7 @@ export class DropdownMenu extends BasePlugin {
 
     if (this.hot.getSettings().ariaTags) {
       setAttribute(button, [
-        A11Y_HIDDEN(),
+        A11Y_LABEL(this.hot.getTranslatedPhrase(COLUMN_HEADER_DESCRIPTION_OPEN_MENU)),
```

Comment:
I'm unable to focus this button with NVDA, so I'm not sure if this is the best way to make the dropdown menu button screen reader compatible.